feature-libs/pickup-in-store/root/components/container/pickup-in-store-order-consignment/pickup-in-store-order-consignment-container.component.ts

```diff
@@ -19,37 +20,19 @@ export type IOutletContextData = { item: Consignment; order: Order };
   selector: 'cx-pickup-in-store-order-consignment',
   templateUrl: './pickup-in-store-order-consignment-container.component.html',
 })
-export class PickupInStoreOrderConsignmentContainerComponent
-  implements OnInit, OnDestroy
-{
+export class PickupInStoreOrderConsignmentContainerComponent implements OnInit {
   constructor(
-    @Optional()
-    protected outlet: OutletContextData<IOutletContextData>
+    @Optional() protected outlet: OutletContextData<IOutletContextData>
   ) {}
-  consignment: Consignment;
-  order: Order;
-  subscription: Subscription = new Subscription();
-  ngOnInit(): void {
-    this.subscription.add(
-      this.outlet?.context$
-        ?.pipe(
-          tap((context) => {
-            this.consignment = context.item;
-            this.order = context.order;
-          })
-        )
-        .subscribe()
-    );
-  }
-  normalizeFormattedAddress(formattedAddress: string): string {
-    const addresses = formattedAddress
-      ?.split(',')
-      ?.map((address) => address.trim());
 
-    return addresses?.filter(Boolean).join(', ');
-  }
+  pointOfService$: Observable<PointOfService>;
```

Comment:
maybe make is as `@Input()`. So, it can be used as child component as well.

Reply:
Added. Thanks